src/js/pages/Sync/index.js

```diff
@@ -81,7 +108,7 @@ export default class Sync extends React.PureComponent {
   componentWillUnmount() {
     this._mounted = false;
     if (this.state.connected && this._socket) {
-      this._socket.disconnect();
+      //this._socket.disconnect();
```

Comment:
Do we not want this?

Reply:
If we disconnect the socket, we are unable to reconnect to same sync code again, or connect from any other instance of sttm-web with that code. We have to stop and start a new session from the desktop. So, I commented it out for now.

Reply:
Also, the link to sync is not yet exposed. So, we aren't planning to release it with this version. It was merged to dev, because of the ikmanzil demo we have. So that they can use dev link to test it out.

Reply:
Here we disconnect only when user leaves the screen. When the screen is reopened, we'll remount.

I understand you might want user to persist the connection, but if we keep it on it might kill the battery on phone and will leave too many open connections on server. Can we instead disconnect it after, say 5 minutes? While persistent connections are cool, they might not scale for our server implementation

Reply:
Yeah, I got that. Even with this I still wasn't able to implement a persistent connection though.

And earlier, if we disconnect the socket from the web, it also disconnected the desktop session. But, I tested it again now, and it doesn't act that way. So uncommenting it now.

Reply:
@bogas04 veerji I am a little confused about this disconnect code. Will continue this conversation on slack. I have commented the code again, after uncommenting it.

Reply:
any update here?